### DISCUSSION TITLE
Archer T4U V.1

### DISCUSSION THREAD

**Discussion by cripsx** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#144] [Ideas]
Hey Chris, Thank you for your support, your code works on an Archer T4U V.1 it's the first generation of TP-Link Archer´s, thought I will have to use it only on my Windows Computer, but now I can use for my Macbook to, thx. I'm also using the most recent Version of MacOs, Monterey 12.0.1

**Comment by chris1111** [comment]
Thanks I will indicated  the support for the V1 and V2
